pkg/gardenlet/controller/shoot/shoot_control_delete.go

```diff
@@ -190,6 +190,11 @@ func (c *Controller) runDeleteShootFlow(o *operation.Operation, errorContext *er
 			Fn:   flow.TaskFn(botanist.SyncClusterResourceToSeed).RetryUntilTimeout(defaultInterval, defaultTimeout),
 		})
 
+		_ = g.Add(flow.Task{
```

Comment:
Just to double check: Is it correct that no other task is depending on this one?

Reply:
Yep, I will add the dependant tasks with following PRs. Same for below.